web/src/main/java/org/cbioportal/web/MrnaPercentileController.java

```diff
@@ -34,9 +34,9 @@
     private MrnaPercentileService mrnaPercentileService;
 
     @RequestMapping(value = "/genetic-profiles/{geneticProfileId}/mrna-percentile/fetch", method = RequestMethod.POST,
-        produces = MediaType.APPLICATION_JSON_VALUE)
+        consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
     @ApiOperation("Get mRNA expression percentiles for list of genes for a sample")
-    public ResponseEntity<List<MrnaPercentile>> getMrnaPercentile(
+    public ResponseEntity<List<MrnaPercentile>> fetchMrnaPercentile(
```

Comment:
No, just a small method name change in Mrna Percentile API that wouldn't worth creating a separate PR.